app/src/main/res/xml/player_settings.xml

```diff
@@ -85,6 +85,13 @@
             app:key="picture_in_picture"
             app:title="@string/picture_in_picture" />
 
+        <SwitchPreferenceCompat
```

Comment:
We should reorder the player preferences since the option gets kind of lost there as it's rather on the bottom while some unimportant settings like the default playback speed or the default resize mode are on the top. I'll probably change that later, just wanted to mention it here.

Reply:
Yes, some options can be moved below which are obvious and users generally don't use